src/scss/docs/bootlaterus-docs.scss

```diff
@@ -4,34 +4,9 @@ h1 {
   margin-bottom: 4.0rem;
 }
 
-.background {
-  top: 0;
-  position: fixed;
-  width: 100%;
-  height: 100vh;
-  z-index: -1;
-
-  &-content {
-    background-size: cover;
-    background-position: center center;
-    background-repeat: no-repeat;
-    position: absolute;
-    top: 0;
-    left: 0;
-    height: 100vh;
-    width: 100%;
-    content: '';
-  
-    &:after {
-      width: 100%;
-      height: 100vh;
-      top: 0;
-      left: 0;
-      position: absolute;
-      background: linear-gradient(transparentize($body-bg, 0.14), $body-bg);
-      content: '';
-    }
-  }
+h2, h3, h4, h5, h6,
+.h2, .h3, .h4, .h5, .h6 {
+  border-bottom: $headings-bottom-border;
```

Comment:
Why are you replacing the border-bottom to all h* if you already have a class for this?

Reply:
Let's use it! Maybe an @extends?